go.mod

```diff
@@ -3,13 +3,23 @@ module github.com/Boeing/config-file-validator
 go 1.21
 
 require (
```

Comment:
Changes to the go.mod file were triggered by `[go mod tidy](https://go.dev/ref/mod#go-mod-tidy)`.